cmd/utils/utils.go

```diff
@@ -180,18 +180,33 @@ func UpdateMILogger(loggerName, loggingLevel string) string {
 	body["loggerName"] = loggerName
 	body["loggingLevel"] = loggingLevel
 
+	if headers[HeaderAuthorization] == "" {
+		headers[HeaderAuthorization] = HeaderValueAuthPrefixBearer + " " +
+			RemoteConfigData.Remotes[RemoteConfigData.CurrentRemote].AccessToken
+	}
+
 	resp, err := InvokeUPDATERequest(url, headers, body)
 
 	if err != nil {
-		HandleErrorAndExit("Unable to connect to host", nil)
+		HandleErrorAndExit("Unable to connect to " + url, nil)
```

Comment:
Can't we pass the error here? Also does this handle method do both printing the error in the server side and responding to the client with the error?